packages/desktop-client/src/components/sidebar/Accounts.tsx

```diff
@@ -64,7 +66,7 @@ export function Accounts() {
       targetIdToMove = idx < accounts.length ? accounts[idx].id : null;
     }
 
-    dispatch(moveAccount(id, targetIdToMove));
+    dispatch(moveAccount({ id, targetId: targetIdToMove as string }));
```

Comment:
Why is the cast needed here? Can we either check the type of the `unknown` before passing it (and error if it's not a string) or just type it as a string?

Reply:
I checked the callers of the moveAccount account and the targetId is always a string so it we should definitely update the type to a string, but let's do it in another PR instead.